_data/projects/racy-game.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: racy-game
+desc: A Racing Car game with Python
+site: https://github.com/shawnfrost80/racy-game
+tags:
+- python
+- pygame
+- numpy
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/amcquade/quick-feed/labels/up-for-grabs
```

Comment:
@shawnfrost80 this link is a different repository than the one in `site` link above. Is this intentional?